src/proxysql.rs

```diff
@@ -201,23 +207,24 @@ impl ProxySQL {
                 );
                 messages::print_note(
                     format!(
-                        "Server HG: {}, Host: {}, Port: {} is currently {}. Changing to {}",
+                        "Server HG: {}, Host: {}, Port: {} is currently {} on proxysql and {} on readyset. Changing to {}",
                         self.readyset_hostgroup,
                         host.get_hostname(),
                         host.get_port(),
-                        host.get_status(),
+                        host.get_proxysql_status(),
+                        host.get_readyset_status().to_string().to_uppercase(),
                         status
                     )
                     .as_str(),
                 );
-                host.change_status(status);
+                host.change_proxysqlstatus(status);
```

Comment:
nit: "change_proxysql_status" for consistency.

Reply:
done